docs/WizardExample.jsx

```diff
@@ -11,13 +11,20 @@ class AddressStep extends React.Component {
   render() {
     const {setWizardState, wizardState, recipient} = this.props;
     return (
-      <form ref={(f) => { this.form = f; }} onSubmit={(e) => e.preventDefault()}>
+      <form
+        ref={f => {
+          this.form = f;
+        }}
+        onSubmit={e => e.preventDefault()}
+      >
```

Comment:
Ah must be the eslint/vscode auto fix on save setting I was using :P i'll try to go through and undo some of this stuff